### PR TITLE
Regroup all editable files in one place

### PR DESCRIPTION
**Pull request by bastienwirtz**
## Description

Regroup all editable files into one place, the `assets` directory. This makes things easier for docker installation, only one directory have to be binded. This change is retro compatible.

what's done: 
- Moved the `config.yml` to `assets`
- Fixed default assets copy

Fixes https://github.com/bastienwirtz/homer/issues/83

### CODE REVIEW

**Comment by bastienwirtz** [comment]
Hey @GlennToms! 
I made a few adjustments on your recent changes, it would be awesome if you can have an have a look!

PS: Docker hub auto build are temporarily disabled, your previews changes to the docker files are not released on docker hub yet.

File: entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 
-yes n | cp -i /www/config.yml.dist /www/config.yml
-while true; do echo n; done | cp -Ri /app/dist/www/assets /www/assets 2>/dev/null
+while true; do echo n; done | cp -Ri /www/default-assets/* /www/assets/
+
```
Comment (GlennToms) [inline entrypoint.sh:4]: We need a way to launch a one of the default configs
You might want to swap out for **config.yml.dist.sample-sui** depending on what you want as the default page when a user starts on the container for the first time.

```suggestion
# This makes sure that at least the default config is available.
yes n | cp -i /www/assets/config.yml.dist /www/assets/config.yml
```

File: Dockerfile.arm64v8 [hunk 2]
```diff
@@ -33,8 +33,9 @@ RUN addgroup -S ${GROUP} -g ${GID} && adduser -D -S -u ${UID} ${USER} ${GROUP} &
 
 COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist /www/
 COPY --chown=${USER}:${GROUP} entrypoint.sh /entrypoint.sh
+RUN mv /www/assets /www/default-assets
```
Comment (GlennToms) [inline Dockerfile.arm64v8:36]:
```suggestion
Replace with `COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist/assets /www/default-assets`

File: Dockerfile.arm32v7 [hunk 3]
```diff
@@ -33,8 +33,9 @@ RUN addgroup -S ${GROUP} -g ${GID} && adduser -D -S -u ${UID} ${USER} ${GROUP} &
 
 COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist /www/
 COPY --chown=${USER}:${GROUP} entrypoint.sh /entrypoint.sh
+RUN mv /www/assets /www/default-assets
```
Comment (GlennToms) [inline Dockerfile.arm32v7:36]:
```suggestion
Replace with `COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist/assets /www/default-assets`

File: Dockerfile [hunk 4]
```diff
@@ -23,8 +23,9 @@ RUN addgroup -S ${GROUP} -g ${GID} && adduser -D -S -u ${UID} ${USER} ${GROUP} &
 
 COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist /www/
 COPY --chown=${USER}:${GROUP} entrypoint.sh /entrypoint.sh
+RUN mv /www/assets /www/default-assets
```
Comment (GlennToms) [inline Dockerfile:26]: We can't assume that **/www/assets** will be accessible to the UID that is running the build. On my system I get **access denied**

```suggestion
Replace with `COPY --from=build-stage --chown=${USER}:${GROUP} /app/dist/assets /www/default-assets`

**Pull request merged**